main.nginx.conf

```diff
@@ -68,6 +68,44 @@ http {
       proxy_set_header Host $host;
     }
 
+    # Enketo URL redirection.
+    # We want the Enketo URLs redirected to a Central frontend page which dynamically decides
+    # whether to show a WebForm or an iframed Enketo.
+    # For that iframe to work, we'll need another path prefix under which we can reach
+    # Enketo — this one will not be intercepted.
+    location /enketo-passthrough {
+      proxy_pass http://localhost:8005/-;
+      proxy_redirect off;
+      proxy_set_header Host $host;
+    }
+
+    location ~* "^/-/(?<enketoPath>single|X)/(?<enketoId>[a-zA-Z0-9]{31})" {
+      # Form fill link, public
+      if ($request_method = GET) {
```

Comment:
I think this if condition is not necessary because we only send GET request to this location